Apps/Src/FaultState.c

```diff
@@ -17,19 +17,19 @@ static void ArrayMotorKill(void) {
 }
 
 void EnterFaultState(void) {
-    if(FaultBitmap.Fault_OS){
+    if(FaultBitmap & FAULT_OS){
         ArrayMotorKill();
     }
-    else if(FaultBitmap.Fault_TRITIUM){
+    else if(FaultBitmap & FAULT_TRITIUM){
         ArrayMotorKill();
     }
-    else if(FaultBitmap.Fault_READBPS){
+    else if(FaultBitmap & FAULT_READBPS){
         ArrayMotorKill();
     }
-    else if(FaultBitmap.Fault_UNREACH){
+    else if(FaultBitmap & FAULT_UNREACH){
         ArrayMotorKill();
     }
-    else if(FaultBitmap.Fault_DISPLAY){
+    else if(FaultBitmap & FAULT_DISPLAY){
```

Comment:
Why not combine into something like
```C
if (FaultBitmap & (FAULT_OS | FAULT_TRITIUM | FAULT_READBPS | FAULT_UNREACH)) {
    ArrayMotorKill();
} else if (FaultBitmap & FAULT_DISPLAY) {
    . . .
}
```

Reply:
Eventually we may need different behavior for each of those faults, not just killing the motor. I created the if statements to distinguish between cases.